google/cloud/forseti/services/inventory/storage.py

```diff
@@ -906,6 +907,32 @@ def get_latest_inventory_index_id(cls, session):
             inventory_index.id)
         return inventory_index.id
 
+    @classmethod
+    def get_inventory_index_id_by_scanner(cls, session, scanner_index_id):
```

Comment:
when the user runs `forseti notifier run --scanner_index_id XXXXX`, the inventory_index_id is not provided, but will be needed to complete the notifier. This function serves to fill the gap